qml/Aqt/Cassowary/Stay.qml

```diff
@@ -23,6 +23,6 @@ import Aqt.Cassowary 1.0
 
 Edit {
     id: edit
-    suggested: actualTarget.value
+    suggested: actualTarget ? actualTarget.value : NaN
```

Comment:
It gets into the solver and then it starts doing weird things. Same happens with `Infinity` (we found out through experience to our disgrace...)